src/main/core/scheduler/scheduler_policy_host_steal.c

```diff
@@ -42,6 +43,7 @@ struct _HostStealThreadData {
     /* which worker thread this is */
     guint tnumber;
     GMutex lock;
+    volatile bool isStealable;
```

Comment:
I don't think you need `volatile` here; the `__atomic` helpers should already tell the compiler not to optimize away those accesses. (OTOH I suppose it doesn't hurt anything other than being slightly confusing why it's being used here)

Reply:
I'll remove it. I don't have much experience with atomics, so this was just me being stupid.